nodejs/lancedb/arrow.ts

```diff
@@ -751,6 +751,19 @@ function makeVector(
   // biome-ignore lint/suspicious/noExplicitAny: skip
 ): Vector<any> {
   if (type !== undefined) {
+    if (DataType.isBool(type)) {
```

Comment:
Could you add a comment saying this is a workaround for this issue in arrow? That way we can one day remove once this is fixed upstream.


```suggestion
    // Workaround for: https://github.com/apache/arrow-js/issues/68
    if (DataType.isBool(type)) {
```